src/app/shared/autocomplete/autocomplete.component.less

```diff
@@ -0,0 +1,53 @@
+@import "../../variables.less";
+
+.autocomplete-section {
+  width: 40vw;
```

Comment:
I think we can avoid `vw` units and try to use `pixels`

Reply:
Used px instead of vw